tests/console/check_system_info.pm

```diff
@@ -94,8 +94,11 @@ sub check_nodejs_common {
     zypper_call('in nodejs-common | tee /tmp/install_nodejs.log');
 
     assert_script_run('grep -E "Installing: nodejs16" /tmp/install_nodejs.log');
-    my $ret_s1 = script_output(q(zypper se -i nodejs | sed -n '/nodejs16 /p' | awk '{print $1}'));
-    my $ret_s2 = script_output(q(zypper se -i nodejs | sed -n '/nodejs-common/p' | awk '{print $1}'));
+    script_run('zypper se -i nodejs');
+    my $ret_s1 = script_output(q(zypper se -i nodejs|grep package|grep nodejs16|awk -F\| '{print $1}'));
+    my $ret_s2 = script_output(q(zypper se -i nodejs|grep package|grep nodejs-common|awk -F\| '{print $1}'));
+    diag "'ret_s1=$ret_s1'";
+    diag "'ret_s2=$ret_s2'";
```

Comment:
Do we still need those information to output?

Reply:
I can remove that, but once it fails like ppc64le, we can know what's wrong and save some time.

Reply:
now moved them to die and will print once it's fails